src/hooks/core/websocket/types.d.ts

```diff
@@ -27,7 +27,10 @@ export type GetstreamMessage = {
   text: string;
   isSystem?: boolean;
   textOwnMessage?: string;
+  reply_data?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `reply_data` property is of type `any`. This can lead to potential issues as it doesn't provide any type checking or autocompletion benefits. If possible, replace `any` with a more specific type that describes the shape of `reply_data`.
```diff
-  reply_data?: any;
+  reply_data?: ReplyDataType; // Replace ReplyDataType with the actual type
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->